rootdir/root/init.qcom.rc

```diff
@@ -460,7 +424,6 @@ service qmuxd /system/bin/qmuxd
 
 service netmgrd /system/bin/netmgrd
     class main
-    disabled
```

Comment:
The problem is bigger than that. I'll take a look at why init.environ.rc makes Android container crash at all. Qcom's init system is driven by a number of shell script, and patching all of them is not a solution. Besides, there's probably more things that rely on the correct environment variable being set.